tests/test_api.py

```diff
@@ -75,11 +75,12 @@ def test_manifest_json(helpers):
     )
     yield test_manifest_path
 
-
-@pytest.fixture(scope="class")
-def data_model_jsonld():
-    data_model_jsonld = "https://raw.githubusercontent.com/Sage-Bionetworks/schematic/develop/tests/data/example.model.jsonld"
-    yield data_model_jsonld
+@pytest.fixture(scope="class",
```

Comment:
@lakikowolfe Thanks for the great work here!  As noted below, I really like this solution.  I just wanted to understand your rationale behind  why you chose this over adding these to `pytest.parametrize`

Reply:
hmm I don't have a rationale except for when I was looking into fixtures (I'm new to pytest) I came across this method. I don't know the difference between the two but will look into it tomorrow.

Reply:
No worries, a quick browse of this documentation: https://docs.pytest.org/en/7.1.x/how-to/parametrize.html and some of the tests below may provide some guidance. 

That said, this was more of a self-learning exercise.  A quick summary is that it would've required more code changes, but I'll leave it to you to see why.